<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBoxDialog</class>
 <widget class="QDialog" name="ToolBoxDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>32</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <widget class="QPushButton" name="ID_BTN_CURSOR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/cursor.png</normaloff>res/cursor.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ID_BTN_DRAW">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>1</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/pen.png</normaloff>res/pen.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ID_BTN_SELECT">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>1</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/select.png</normaloff>res/select.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>31</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>1</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ID_BTN_MOVE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/move.png</normaloff>res/move.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
